docs/core/tutorials/top-level-templates.md

```diff
@@ -101,7 +101,7 @@ You can also add a [`<Using>`](../project-sdk/msbuild-props.md#using) item with
 While a .NET 6 console app template generates the new style of top-level statements programs, using .NET 5 doesn't. By creating a .NET 5 project, you'll receive the old program style. Then, you can edit the project file to target .NET 6 but retain the old program style for the *Program.cs* file.
 
 > [!IMPORTANT]
-> Creating a project that targets .NET 5 requires the .NET 5 templates. The .NET 5 templates can be installed manually with the [`dotnet new --install` command](../tools/dotnet-new-sdk-templates.md#console) or by [installing the .NET 5 SDK](https://dotnet.microsoft.com/download/dotnet/5.0).
+> Creating a project that targets .NET 5 requires the .NET 5 templates. The .NET 5 templates can be installed manually with the [`dotnet new install` command](../tools/dotnet-new-sdk-templates.md#console) or by [installing the .NET 5 SDK](https://dotnet.microsoft.com/download/dotnet/5.0).
```

Comment:
@tdykstra please advice what should we do with this article?

Starting 6.0.300 there is a switch to disable top-level statements (`--use-program-main`), should we document it instead?
This chapter refers to .NET 5, which is EOL.

Reply:
In that doc we need to delete the content that recommends targeting .NET 5 and replace it with the ---program-main option or corresponding Visual Studio checkbox.

Reply:
i did my best to update it, could you please check it?